packages/clerk-js/src/core/resources/ExternalAccount.ts

```diff
@@ -52,6 +53,7 @@ export class ExternalAccount extends BaseResource implements ExternalAccountReso
     this.providerUserId = data.provider_user_id;
     this.approvedScopes = data.approved_scopes;
     this.avatarUrl = data.avatar_url;
+    this.experimental_imageUrl = data.image_url;
```

Comment:
❓ Why did we decided to append `experimental_` in our resource classes  ?

Reply:
I used the same approach as in `User`. This is a way to define an experimental property (and discourage users from using it) suggested by @nikosdouvlis . We will replace the property with the `imageUrl` when the feature of clerk images is completely released.